### ISSUE TITLE
UWP support?

### ISSUE THREAD

**Issue by ty-dev** [xinthink/react-native-material-kit#193]
Does this work for UWP apps?

**Comment by Crash--** [comment]
I guess not. We only have native code for Android and iOS at the moment. 

If WUP becomes a real RN functionality maybe we'll think about adding it to RNMK. If you really want it, you can help up by submitting PRs ;).

**Comment by semiautomatix** [comment]
Yeah, gonna have to skip this module as a result - since we're note targeting Windows.
